src/main.ts

```diff
@@ -9,6 +9,7 @@ import { createRouter, createWebHistory } from 'vue-router'
 import 'uno.css'
 import './styles/soramitsu-ui.scss'
 import './styles/main.sass'
+import 'github-markdown-css/github-markdown-light.css'
```

Comment:
Maybe move it to `Proposal/Description.vue`?

Reply:
Yeah, sure

Reply:
Done